---
content_type: resource
description: 'This resource covers: Pairwise Alignment, Dot Plots, Two possible models,
  A Probabilistic Model of Alignment, The Odds Ratio Statistic, Substitution Matrix,
  Needleman-Wunsch(global), Smith-Waterman (Local Alignment), Classical Approach:
  Extreme Value Distribution, Building Phylogenetic Trees, Parsimony-based Phylogenetic
  Trees, etc.'
file: /courses/7-90j-computational-functional-genomics-spring-2005/89b3b48dc4dcf04b45301fe925ecfcd8_lecture_2.pdf
file_size: 244991
file_type: application/pdf
learning_resource_types:
- Lecture Notes
license: https://creativecommons.org/licenses/by-nc-sa/4.0/
ocw_type: OCWFile
parent_title: Lecture Notes
parent_type: CourseSection
parent_uid: 8e422c8f-7248-ab9d-9c08-c848e8f61f13
resourcetype: Document
title: lecture_2.pdf
uid: 89b3b48d-c4dc-f04b-4530-1fe925ecfcd8
video_files:
  video_thumbnail_file: null
video_metadata:
  youtube_id: null
---
This resource covers: Pairwise Alignment, Dot Plots, Two possible models, A Probabilistic Model of Alignment, The Odds Ratio Statistic, Substitution Matrix, Needleman-Wunsch(global), Smith-Waterman (Local Alignment), Classical Approach: Extreme Value Distribution, Building Phylogenetic Trees, Parsimony-based Phylogenetic Trees, etc.